### ISSUE TITLE
In the Securing your GraphQL API documentation, the introductory sentence is difficult to read

### ISSUE THREAD

**Issue by dthyresson** [graphql-hive/envelop#510]
**Describe the bug**

In the Securing your GraphQL API [documentation](https://www.envelop.dev/docs/guides/securing-your-graphql-api) the introductory sentence is difficult to read:

> Building a secure GraphQL API is hard by design of to the because "Graph" part withing GraphQL. 

**To Reproduce**

* Visit: https://www.envelop.dev/docs/guides/securing-your-graphql-api
* Read intro paragraph

**Expected behavior**

Perhaps reword this to:

```
Building a secure GraphQL API is hard by design because of the "Graph" nature of GraphQL.
```

**Comment by n1ru4l** [comment]
Thank you 🙏
